tests/t5/test_tokenization_t5.py

```diff
@@ -223,7 +223,7 @@ def test_outputs_not_longer_than_maxlen(self):
             ["I am a small frog" * 1000, "I am a small frog"], padding=True, truncation=True, return_tensors=FRAMEWORK
         )
         self.assertIsInstance(batch, BatchEncoding)
-        self.assertEqual(batch.input_ids.shape, (2, 512))
+        self.assertEqual(batch.input_ids.shape, (2, 8001))
```

Comment:
@sgugger note that while this IMO fixes a bug (T5 has no fixed max length), it might break backwards compatibility in some edge cases. T5 is used a lot, but still think it's better to correct it here.